vars/ciMetrics.groovy

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Should this class reside in src/ ?

Reply:
Yes, but I was unsure how to make that work. The class is used to collect data throughout the entire build and then send it to influx after the build completes, so there can only be one instance of it. I'm seeing now though that there's a @Singleton annotation in groovy that can be used. I'll do some testing with that.